packages/account/src/BiconomySmartAccount.ts

```diff
@@ -286,6 +294,9 @@ export class BiconomySmartAccount extends SmartAccount implements IBiconomySmart
       callData: callData
     }
 
+    // for this Smart Account dummy ECDSA signature will be used to estimate gas
+    userOp.signature = this.getDummySignature()
```

Comment:
but even if sdk doesn’t send a dummy signature the bundler still appends a dummy signature right? do we need to add this @talhamalik883

Reply:
bundler wouldn't know account information. I think it accepts a dummy sig and if not present defaults to ECDSA mock signature.